index.js

```diff
@@ -76,10 +74,10 @@ function resolveAndLoadModules(ids, opts, cb) {
 }
 
 module.exports = function(filename, opts) {
-  if (!isSJS.exec(filename))
-    return through();
-    
   opts = opts || {};
+  opts.extensions = opts.extensions || /.+\.sjs$/;
```

Comment:
That would work when configuring `sweetify` from Node.js but won't if you specify options through `package.json` or command line.